server/lib/payments.js

```diff
@@ -555,7 +555,7 @@ export const executeOrder = async (user, order, options = {}) => {
   if (!order) {
     return Promise.reject(new Error('No order provided'));
   }
-  if (order.processedAt) {
+  if (order.processedAt && !options.isAddedFund) {
```

Comment:
Maybe add a small comment there to explain this check?

Reply:
I've added a small comment. Let me know if that looks good.